### PR TITLE
Adjust documentation to globaldb v3

### PR DESCRIPTION
**Pull request by nebolax**

### CODE REVIEW

File: readme.MD [hunk 1]
```diff
@@ -28,44 +28,44 @@ Make sure that the `updates/info.json` file is always kept up to date with the u
 
 Only the last one needs to be done manually. The rest can be taken care of by running the ``populate_infojson.py`` script.
 
-### Adding ethereum tokens
+### Adding evm tokens
 
 ```
-INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x6B175474E89094C44Da98b954EedeAC495271d0F", 8, "maker");INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x6B175474E89094C44Da98b954EedeAC495271d0F", "C", "New Multi Collateral DAI", "NDAI", 1573672677, NULL, "dai", NULL, "0x6B175474E89094C44Da98b954EedeAC495271d0F");
+INSERT INTO assets(identifier,type, started, swapped_for) VALUES("eip155:1/erc20:0xC2FEC534c461c45533e142f724d0e3930650929c", "C", 123, NULL); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0xC2FEC534c461c45533e142f724d0e3930650929c", "A", "A", "0xC2FEC534c461c45533e142f724d0e3930650929c", 18, NULL); INSERT INTO common_asset_details(identifier, name, symbol, coingecko, cryptocompare, forked) VALUES("eip155:1/erc20:0xC2FEC534c461c45533e142f724d0e3930650929c", "AKB token", "AKB", NULL, "AIDU", NULL);
```
Comment (yabirgb) [inline readme.MD:34]: Can you maybe change it to a token where the coingecko field is not NULL?

**Review by yabirgb — COMMENTED** [review]
Just one small comment

File: readme.MD (hunk 1 above)
Comment (nebolax) [inline readme.MD:34]:
```suggestion
INSERT INTO assets(identifier,type, started, swapped_for) VALUES("eip155:1/erc20:0x422866a8F0b032c5cf1DfBDEf31A20F4509562b0", "C", 123, NULL); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0xC2FEC534c461c45533e142f724d0e3930650929c", "A", "A", "0xC2FEC534c461c45533e142f724d0e3930650929c", 18, NULL); INSERT INTO common_asset_details(identifier, name, symbol, coingecko, cryptocompare, forked) VALUES("eip155:1/erc20:0x422866a8F0b032c5cf1DfBDEf31A20F4509562b0", "Adshares", "ADST", "adshares", "ADS", NULL);
```

Comment (nebolax) [reply readme.MD]: Done. Through github UI interface! :smile:

**Pull request merged**